content/source/docs/enterprise/vcs/index.html.md

```diff
@@ -14,6 +14,19 @@ Terraform Enterprise (TFE) is more powerful when you integrate it with your vers
 
 We recommend configuring VCS access when first setting up a TFE organization, and you might need to add additional VCS providers later depending on how your organization grows.
 
+## Supported VCS Providers
+
+For details on configuring VCS access, select a supported VCS provider from the list below:
```

Comment:
```suggestion
TFE supports the following VCS providers:
```

Reply:
Hm...I was trying to keep in a note that you can click those links to get detailed instructions. Do you think that's unnecessary?

Reply:
@nfagerlund want to take another look? I changed the intro line to match yours, and added the other info below.